.gitignore

```diff
@@ -91,3 +91,7 @@ ui-react/syndesis/.env.production.local
 # build cache
 ui-react/*/.cache
 ui-react/*/.rts2_cache*
+
+# package-lock.json
+*/ui-react/package-lock.json
```

Comment:
Good call but one should never ever user `npm` with Syndesis, only `yarn`

Reply:
better be safe than sorry ;)